### ISSUE TITLE
JSON serialization does not need to be pretty-printed

### ISSUE THREAD

**Issue by mobitar** [standardnotes/android#53]
I'm looking at the way JSON comes in from a change, and it looks like it's pretty printed (has a lot of newlines). I don't think we need this. Might save some space.

<img width="861" alt="screen shot 2017-02-19 at 9 44 46 am" src="https://cloud.githubusercontent.com/assets/3277844/23103870/32017e5a-f688-11e6-8d59-07a58369cf3a.png">

It should however be pretty-printed for when the user manually exports their data to email.

**Comment by deftelf** [comment]
Yes it was changed for the export and I dont' think it was necessary to have 2 exporters for the amount of data we do - but yes this wants fixing

**Comment by mobitar** [comment]
Ya not a huge deal. Actually just noticed iOS does the same thing 😳
